nixnet/_session/collection.py

```diff
@@ -94,7 +94,7 @@ def __eq__(self, other):
             other_collection = typing.cast(Collection, other)
             return (
                 self._handle == other_collection._handle
-                and self._list_cache == other_collection._list_cache
+                and self._list_cache == other_collection._list_cache  # NOQA: W503
```

Comment:
The code here actually matches our style guide. I would recommend ignoring W503 in the tox.ini in the flake8 section. In the future it would be good to use ni-python-styleguide for linting instead of flake8 directly, but for now if you run into warnings or errors that [ni-python-styleguide ignores](https://github.com/ni/python-styleguide/blob/be4560834ca09e9e1f08532481c841d91ba5cb57/ni_python_styleguide/config.ini#L31), you can just add them to tox.ini.

Reply:
Thanks for the info, I've added W503 to the flake8 ignore list in tox.ini in the latest commit.